code/modules/economy/cash_register.dm

```diff
@@ -546,7 +546,7 @@
 	..()
 
 /obj/machinery/cash_register/cargo
-	account_to_connect = "Cargo"
+	account_to_connect = "Guild"
```

Comment:
This is commented out? Are we planning on making use of this later?

Reply:
yes indeed we are
I didn't activate them in this PR, beyond scope for now, not sure if they'd be bugged. but i'll do it in another future one soonish